exonum/src/blockchain/mod.rs

```diff
@@ -221,7 +221,7 @@ impl Blockchain {
             {
                 let mut schema = Schema::new(&mut fork);
                 if schema.block_hash_by_height(Height::zero()).is_some() {
-                    // TODO create genesis block for MemoryDB and compare in hash with zero block
+                    // TODO create genesis block for MemoryDB and compare in hash with zero block. (ECR-1630)
```

Comment:
`...compare it hash...`

Reply:
Another remark: I did not correct any grammar in existing TODOs (it've already taken too much time, grammar checks would have make this activity too big).
But if you asking me to (: